snapcraft/internal/project_loader/_config.py

```diff
@@ -89,6 +89,7 @@ def __init__(self, project_options=None):
         if project_options is None:
             project_options = snapcraft.ProjectOptions()
 
+        self.build_snaps = set()
         self.build_tools = []
```

Comment:
have I said how much I hate this build_tools? :D
I'm just saying it again to put myself pressure to fix it.